### PR TITLE
Restaurant_Page

### PR DESCRIPTION
**Pull request by cgcarlosg**
This project is the practice for DOM manipulation with JavaScript and Webpack.
This project is A landing page for a restaurant where you can visit.
And You can see the menu with a price.
I built this project with
**- Webpack
VS Code
JavaScript**

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @cgcarlosg  :wave:

__You've done well so far.
However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] There seems to be a conflict with your PR. Kindly fix it.
conflict

- [ ] Kindly deploy the project and add the live demo link to the Readme
- [ ] Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
````
- [ ] You should use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) for all your scripts.
- Use of arrow functions instead of the function keyword

That's it!



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**
